code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -368,7 +367,7 @@
 	min_age = 18
 	icobase = 'icons/mob/human_races/r_vox_old.dmi'
 	deform = 'icons/mob/human_races/r_def_vox_old.dmi'
-	inherent_verbs = list(/mob/living/proc/shred_limb, /mob/living/proc/eat_trash)
+	//inherent_verbs = list(/mob/living/proc/shred_limb, /mob/living/proc/eat_trash)	//Removed trash eater and hardvore traits due to being put into prefs
```

Comment:
That said, prometheans probably shouldn't have shred limb to begin with.

Reply:
As another note, this line is referring to Vox, so I think them having had shred limb makes sense.